graphql/schema/types/task.graphql

```diff
@@ -7,6 +7,28 @@ enum TestSortCategory {
 }
 
 ###### INPUTS ######
+"""
+TestFilterOptions is an input for the task.Tests query.
+It's used to filter, sort, and paginate test results of a task.
+"""
+input TestFilterOptions {
+  testName: String!
+  statuses: [String!]
+  groupID: String!
+  sort: [TestSortOptions!]
+  limit: Int!
+  page: Int!
```

Comment:
Can we make these optional please? We don't always populate them on the UI so I would rather we avoid requiring variables that would always be nil or undefined and avoid forcing consumers of the GraphQL API to provide them.

It should be handled in the resolver. You can give these values defaults instead if you would like them to have a value. But I wouldn't want the client to have to supply the value.

Reply:
Done!